analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -163,7 +169,15 @@ class Bundler(
                 scopeDependencies = scopes.toSortedSet()
             )
 
-            val packages = gemSpecs.values.mapTo(sortedSetOf()) { getPackageFromGemspec(it) }
+            val allProjectDeps = groupedDeps.values.flatten().toSet()
+            val uniqueHelperDeps = HELPER_SCRIPT_DEPENDENCIES.filterNot { it in allProjectDeps }
```

Comment:
When reading `uniqueHelperDeps` I first thought `unque` means `distinct`. But, I believe that it actually means, dependencies which are only helper dependencies but not project dependencies.
How about renaming this to `helperOnlyDependencies` or `dependenciesToRemove`.

Reply:
I'll go for `helperOnlyDeps`.